package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "CivicTechJobs",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Looks like you may have accidentally ran `npm install` the root folder which added this file

Reply:
Oops, removed